src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -63,6 +63,51 @@ static void SendUprobesFullAddressInfoToListener(
   listener->OnAddressInfo(std::move(address_info));
 }
 
+static bool CallstackIsInUserSpaceInstrumentation(
```

Comment:
nit: I would name this `IsCallstackInUserSpaceInstrumentation`. Similar below.

Reply:
You seem to agree on this, but function/method names never ask questions. Rather, a function like this indicates a property. Consider for example the usage in an if (or while):
"if callstack is in user space instrumentation then do something"
vs
"if is callstack in user space instrumentation then do something"
The second one would mix a statement and a question.